src/schema_registry/routers/raw_path_router.py

```diff
@@ -0,0 +1,39 @@
+"""
+Copyright (c) 2025 Aiven Ltd
+See LICENSE for details
+"""
+
+"""
+The MIT License (MIT)
+
+Copyright (c) 2018 Sebastián Ramírez
+"""
+
+from fastapi import HTTPException  # noqa: E402
+from fastapi.routing import APIRoute  # noqa: E402
+from starlette.routing import Match  # noqa: E402
+from starlette.types import Scope  # noqa: E402
+
+import re  # noqa: E402
```

Comment:
The `# noqa` comments are due to double header. Ruff gets confused.

Reply:
Maybe we add to the flake8 config to ignore specific codes per-file, i.e. https://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-per-file-ignores

Reply:
As long as the change is this small I am ok with this, I would not bother to add per file configs.